ClimateMonitorV2/routes/index.js

```diff
@@ -18,6 +18,9 @@ router.get('/', function (req, res) {
 router.get('/about', function (req, res) {
     res.render('about', { title: 'About SheffSenseV2' });
 });
+router.get('/detailedstats', function (req, res) {
+    res.render('detailedstats', { title: 'About SheffSenseV2' });
```

Comment:
Should this title be something like { title: 'SheffSenseV2 Statistics' }

Reply:
Updated with correct title